README.md

```diff
@@ -1,13 +1,7 @@
 # hexo-renderer-jade
 [![Build Status](https://travis-ci.org/hexojs/hexo-renderer-jade.svg?branch=master)](https://travis-ci.org/hexojs/hexo-renderer-jade)  [![NPM version](https://badge.fury.io/js/hexo-renderer-jade.svg)](http://badge.fury.io/js/hexo-renderer-jade) [![Coverage Status](https://img.shields.io/coveralls/hexojs/hexo-renderer-jade.svg)](https://coveralls.io/r/hexojs/hexo-renderer-jade?branch=master) 
 
-**Now Jade is Pug**
-
-Support of [Pug] templates. Made after renaming well-known template engine
-**Jade** for new files with `*.pug` extension.
-
-Backward capapilities is supported, and `*.jade` files is rendered via the old jade engine.
-`*.pug` files will be compiled throught the lastest pug renderer.
+Support [Pug] and Jade templates. `*.pug` and `*.jade` files are rendered using Pug and Jade engine respectively.
```

Comment:
Would you please change to deprecation message after release [hexo-renderer-pug](https://github.com/hexojs/hexo-renderer-pug)?

Reply:
Definitely, I do intend to deprecate the npm package and this repo.

Reply:
Added deprecation message in https://github.com/hexojs/hexo-renderer-jade/pull/35